scalanet/test/src/io/iohk/scalanet/peergroup/kademlia/BitVectorCodecSpec.scala

```diff
@@ -31,6 +31,8 @@ object BitVectorCodecSpec {
   private def encodeDecodeTest[T](implicit codec: Codec[T], a: Arbitrary[T], eq: Equivalence[T]): Unit = {
     forAll(arbitrary[T]) { t =>
       val e = codec.encode(t)
+      System.out.println(t)
```

Comment:
is this an standar practice? If not, and is going to be removed at the end, always include a FIXME  message